Build/README.md

```diff
@@ -0,0 +1,97 @@
+# Continuous Integration for Newtonsoft.Json-for-Unity.Converters
+
+This is how to make it build inside CircleCI. The licenses are needed to be
+added via environment variables inside the project settings within CircleCI.
+
+## Obtain Unity license .ulf files
+
+Repeat the following steps for the following versions:
+
+- 2018.4.14f1
+- 2019.2.11f1
+- 2020.1.0b6-linux-il2cpp
+
+1. Run `Build/local_docker_run.ps1` in PowerShell with `-UnityVersion` parameter
+   being set to Unity version, ex:
+
+   ```powershell
+   # Assuming working directory is at repo root
+   &'.\Build\local_docker_run.ps1' -UnityVersion '2019.2.11f1'
+   ```
+
+   If running Linux or MacOS, you can install [PowerShell Core](https://github.com/PowerShell/PowerShell)
+   to run PowerShell scripts, and then invoke the script like so:
+
+   ```bash
+   # Assuming working directory is at repo root
+   pwsh ./Build/local_docker_run.ps1 -UnityVersion '2019.2.11f1'
+   ```
+
+   If using Git Bash for Windows or WSL, you could also run the script like so:
+   ```bash
+   # Assuming working directory is at repo root
+   powershell.exe ./Build/local_docker_run.ps1 -UnityVersion '2019.2.11f1'
+   ```
+
+2. Enter personal username+password
+
+3. Wait for the log to spit out XML
+
+4. Copy that XML
+
+5. Paste that XML into a new file with the `.alf` extension,
+   ex: `Unity_v2019.2.11f1.alf`
+
+6. Go to <https://license.unity3d.com/manual>
+
+7. Login with the same username+password as mentioned in step 2.
+
+8. Upload the `.alf` file mentioned in step 5.
+
+9. Follow the instructions, for example select "Personal license"
+
+10. Download the new license
+
+11. Place it into the `Build` folder inside this repo
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `1`, was `11`](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5459451)